src/components/ChartView/ChartView.tsx

```diff
@@ -12,11 +12,17 @@ import "./ChartView.css";
 interface IChartViewProps {
   chartID: string;
   fetchChartVersionsAndSelectVersion: (id: string, version?: string) => Promise<{}>;
-  deployChart: (version: IChartVersion, releaseName: string, namespace: string) => Promise<{}>;
+  deployChart: (
+    version: IChartVersion,
+    releaseName: string,
+    namespace: string,
+    values: string,
+  ) => Promise<{}>;
   push: (location: string) => RouterAction;
   isFetching: boolean;
   selected: IChartState["selected"];
   selectChartVersionAndGetReadme: (version: IChartVersion) => Promise<{}>;
+  selectChartVersionAndGetValues: (version: IChartVersion) => Promise<{}>;
```

Comment:
we're selecting the chart version twice, so we should probably have a single action called `selectChartVersionAndGetFiles` that dispatches both `getReadme` and `getValues`. Or, we simply have a `getValues` action and avoid selecting the ChartVersion twice.